src/components/common/ActionModal.vue

```diff
@@ -262,6 +268,16 @@ export default {
       }
       return signWithLocalKeystore
     },
+    browserWithLedgerSupport() {
```

Comment:
I would extract this function, and use it here and in `TmSessionWelcome`.  Best not to repeat it.

Reply:
good call! @migueog you can move it to `session.js`

Reply:
@colw Ah great thanks!

@jbibla It has been moved! Thanks for the help